src/libgrate/grate-3d.c

```diff
@@ -659,11 +659,13 @@ static void grate_3d_set_texture_desc(struct host1x_pushbuf *pb,
 				      struct host1x_pixelbuffer *pixbuf,
 				      unsigned max_lod,
 				      unsigned wrap_mode,
-				      bool mip_filter,
-				      bool mag_filter,
-				      bool min_filter)
+				      bool mipmap_enabled,
+				      bool min_filter_linear_within,
+				      bool min_filter_linear_between,
+				      bool mag_filter_linear)
```

Comment:
I'm not used to any naming convention, so do not really care. Just tell me how to name them.

MIPFILTER == MINFILTER_LINEAR_WITHIN ?
MINFILTER == MINFILTER_LINEAR_BETWEEN ?

Is that ^ correct?